recipes/gklib/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "gklib" %}
+{% set version = "5.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KarypisLab/GKlib/archive/METIS-v{{ version }}-DistDGL-0.5.tar.gz
+  sha256: 52aa0d383d42360f4faa0ae9537ba2ca348eeab4db5f2dfd6343192d0ff4b833
+
+build:
+  number: 0
+  # skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake >=3.15
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+
+test:
+  commands:
+
+    - test -f ${PREFIX}/bin/cmpnbrs  # [unix]
```

Comment:
Is it possible to try executing any of these? Even a simple `cmpnbrs --version` or `cmpnbrs --help` can catch a lot of potential mistakes.

Reply:
Sure the `--help` commands should be do-able! @hadim do you want to add these, or if you give me write access on your fork, I'd be happy to!

Reply:
Thanks @SimonBoothroyd. I just gave you write access to my fork so you can push into this PR.

Reply:
Thanks - tests added in 1342d88.